conf/cassandra.yaml

```diff
@@ -1805,3 +1805,10 @@ drop_compact_storage_enabled: false
 #    heartbeat_file: /var/lib/cassandra/data/cassandra-heartbeat
 #    excluded_keyspaces: # comma separated list of keyspaces to exclude from the check
 #    excluded_tables: # comma separated list of keyspace.table pairs to exclude from the check
+
+sstable_compressor:
+#  Defines the compression params used to read/write the sstables.  If undefined the old default is used.
```

Comment:
```suggestion
#  Defines the compression params used to read/write the sstables.  If undefined the default is LZ4 with 16kb chunk length. To disable compression set `class_name: None`
```

And `class_name: None` doesn't look to be implemented yet…

Reply:
true! edited previous comment.

Reply:
nit: sstables -> SSTables